src/utils/helperMethods.ts

```diff
@@ -0,0 +1,15 @@
+export function findFileCountOfJSConversionsToTS(filesAdded: string[], filesRemoved: string[]) {
+  let count = 0;
+  filesAdded.forEach((d) => {
+    const splitFileName = d.split('.');
+    filesRemoved.forEach((e) => {
```

Comment:
Might be able to use a `filesRemoved.find(<find matching file>)` here to avoid a nested loop